test/e2e/common/node/container_probe.go

```diff
@@ -519,7 +519,7 @@ var _ = SIGDescribe("Probing container", func() {
 		Testname: Pod liveness probe, using grpc call, success
 		Description: A Pod is created with liveness probe on grpc service. Liveness probe on this endpoint will not fail. When liveness probe does not fail then the restart count MUST remain zero.
 	*/
-	ginkgo.It("should *not* be restarted with a GRPC liveness probe [NodeAlphaFeature:GRPCContainerProbe][Feature:GRPCContainerProbe]", func() {
```

Comment:
I don't see these tests running in the presubmit e2es... do they need the `[Feature:GRPCContainerProbe]` tag removed?

Reply:
we used to inconsistently apply Feature for beta features. With https://github.com/kubernetes/enhancements/pull/3042 and in general, I agree, `NodeConformance` works better.